config/Dockerfiles/tests.d/general/01_general

```diff
@@ -3,7 +3,7 @@
 # Verify basic provisioning for all supplied providers
 # distros.exclude: none
 # providers.include: none
-# providers.exclude: none
+# providers.exclude: beaker
```

Comment:
Can we revert this? I'd like beaker to not be excluded

Reply:
Will update that.